app/src/main/AndroidManifest.xml

```diff
@@ -136,7 +136,7 @@
             android:name="com.poupa.vinylmusicplayer.service.MusicService"
             android:enabled="true"
             android:label="@string/app_name"
-            android:exported="false">
+            android:exported="true">
```

Comment:
@AdrienPoupa I let you choose to either explicitly request this to be exported, or omit the line as it was originally

Reply:
Ah yes that makes sense to allow it. Nice catch thanks!